src/Common/src/System/Diagnostics/Debug.Unix.cs

```diff
@@ -8,108 +8,118 @@ namespace System.Diagnostics
 {
     // Intentionally excluding visibility so it defaults to internal except for
     // the one public version in System.Diagnostics.Debug which defines
-    // another version of this partial class with the public visibility 
+    // another version of this partial class with the public visibility
     static partial class Debug
     {
-        private static string NewLine => "\n"; 
+#if !MONO
```

Comment:
The other solution is for `Debug.Windows.cs` (and `Debug.Unix.cs` respectively):
```
static partial class Debug
{
#if MONO
    private static string Windows_NewLine => "\n\r";
#else
    private static string NewLine => "\n\r";
#endif

#if MONO
    private static IDebugLogger Windows_s_logger = new WindowsDebugLogger();
#else
    internal static IDebugLogger s_logger = new WindowsDebugLogger();
#endif

    internal sealed class WindowsDebugLogger : IDebugLogger
    {
        .......
    }
}
```
and in `Debug.Mono.cs` we would have the following code:
```
static partial class Debug
{
    private static string NewLine => PlatformHelper.IsWindows ? Windows_NewLine : Unix_NewLine;

    internal static IDebugLogger s_logger => PlatformHelper.IsWindows ? Windows_s_logger : Unix_s_logger;
}
```